src/data/faq.json

```diff
@@ -65,12 +65,25 @@
                         "anchor": "where_can_i_get_tested",
                         "active": true,
                         "textblock": [
+                            "See also <a href=#test_result_issues>What can I do if I have not received my test result or test certificate?</a>",
                             "<div class='container'><div class='row'><div class='col-sm'><a href='https://map.schnelltestportal.de' target='_blank' rel='noopener'><img class='img-fluid' src='../../assets/img/faq/schnelltestportal_1.jpeg' alt='image'></a></div><div class='col-sm'>Meanwhile, more than 100 rapid test partners are connected to the CWA infrastructure and can provide their test results to the app.<br>A local search that shows the compatible partners with nearby testing sites in your area can be found here: <a href='https://map.schnelltestportal.de' target='_blank' rel='noopener noreferrer'>map.schnelltestportal.de</a><br>The list of test sites contains only those entries that have been reported by the partners concerned.</div></div></div>",
                             "<hr/>",
                             "<div class='container'><div class='row'><div class='col-sm'><a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener'><img class='img-fluid' src='../../assets/img/faq/BMG_testzentren.png' alt='image'></a></div><div class='col-sm'>An overview of the <b>PCR and rapid antigen test sites</b> can be found on this <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG webpage</a> (not clear whether the test site is connected to the CWA or not).</div></div>",
                             "<hr/>",
                             "Another list of test sites from the GitHub community: <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Where to get tested?</a>"
                         ]
+                    },
+                    {
+                        "title": "What can I do if I have not received my test result or test certificate?",
+                        "anchor": "test_result_issues",
+                        "active": true,
+                        "textblock": [
+                            "Results of <b>rapid antigen tests</b> should be transmitted via the CWA within <b>half an hour</b>, results of </b>PCR tests</b> within <b>48 hours</b>.",
+                            "If you have not received your <b>test result</b>, or if the test result has been transmitted but not the <b>digital certificate of the EU</b>, you can report this or other problems as follows:",

```

Comment:
Please check the name used on actual test certificates:

"digital certificate of the EU" sounds like a literal translation of the German.

more likely:
"EU digital COVID test certificate"

Reply:
sure, usually it is just EU digital COVID certificate, however, let's use the word 'test' to make it clearer.